Packages/Tracking/Core/Runtime/Scripts/LeapProvider.cs

```diff
@@ -53,10 +53,12 @@ public abstract class LeapProvider : MonoBehaviour
         /// </summary>
         public abstract Frame CurrentFixedFrame { get; }
 
+        protected TrackingSource _trackingSource;
+
         /// <summary>
-        /// Can be used to provide information as to whether the tracking data source is valid
+        /// Represents the source of tracking data
         /// </summary>
-        public virtual bool CanProvideData { get { return true; } }
+        public virtual TrackingSource TrackingDataSource { get { return _trackingSource; } }
```

Comment:
Makes me wonder if we can write some Unit tests around this....

Reply:
Possibly, though it involves also enabling/disabling various platform services to check different sources